README.md

```diff
@@ -47,6 +47,18 @@ This will open a furniture store complete with all of the props. Select an item
 
 > Note: The place on ground button sometimes does not work properly depending on where the native detects the ground to be.
 
+## Dynamic Doors Preview & Setup
+
```

Comment:
nit: maybe just add a note in here on what is meant by dynamic doors (ie: place doros and that they work like a door, rather than a static asset that cant be open/closed)

Side note for future, it might even be cool to see if we can make some of these interior doors lockable (i wonder if it would work with the existing doorlock systems when you place a door down??)

Reply:
That would be something to look into for the near future I think, That wouuld be pretty sick!